mlflow/server/js/src/experiment-tracking/components/experiment-page/hooks/useRunsArtifacts.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { listArtifactsApi } from '../../../actions';
+
+/**
+ * Fetches artifacts given a list of run UUIDs
+ * @param runUuids List of run UUIDs
+ * @returns Object containing artifacts keyed by run UUID
+ */
+export const useRunsArtifacts = (runUuids: string[]) => {
+  const [artifactsKeyedByRun, setArtifactsKeyedByRun] = useState({});
```

Comment:
we have [ArtifactListFilesResponse](https://github.com/mlflow/mlflow/blob/532381db1f357c18e2d616faf179cf6e35744e15/mlflow/server/js/src/experiment-tracking/types.ts#L382) for this type

```suggestion
  const [artifactsKeyedByRun, setArtifactsKeyedByRun] = useState<Record<string, ArtifactListFilesResponse>>({});
```